### PR TITLE
[feature] Improve storage: Update DBStorage and FileStorage, adding t…

### PR DESCRIPTION
**Pull request by Leidysalda**
this are new update over db_storage and file_storage engines project

modified:   models/engine/db_storage.py
modified:   models/engine/file_storage.py
modified:   tests/test_models/test_engine/test_db_storage.py
modified:   tests/test_models/test_engine/test_file_storage.py

### CODE REVIEW

File: models/engine/db_storage.py [hunk 1]
```diff
@@ -74,3 +74,16 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """ returns the object based on the class name and its ID"""
+        if cls and id:
+            objs = self.all(cls)
+            item = "{}.{}".format(cls.__name__, id)
+            return objs[item]
```
Comment (Leidysalda) [inline models/engine/db_storage.py:83]: I have review your code and find a issues using a unnecessary variable into the function. For example, the item variable is not necessary

File: tests/test_models/test_engine/test_db_storage.py [hunk 2]
```diff
@@ -70,6 +70,13 @@ def test_dbs_func_docstrings(self):
 
 class TestFileStorage(unittest.TestCase):
     """Test the FileStorage class"""
+
+    @classmethod
+    def setUpClass(cls):
+        """Set up for the DBStorage tests"""
+        # cls.state = State()
+        # cls.state.save()
```
Comment (Leidysalda) [inline tests/test_models/test_engine/test_db_storage.py:78]: In this function I noted that you have not functionality and also you could avoid to use the comment using hashtag (#), follow the docstring documentation in python

File: tests/test_models/test_engine/test_db_storage.py [hunk 3]
```diff
@@ -86,3 +93,24 @@ def test_new(self):
...
+    def test_get(self):
+        """ Test if there is exist """
+        # storage = DBStorage()
+        storage = models.storage
+        state = State()
+        state.save()
+        # state.id = '1'
+        obj = storage.get(State, state.id)
+        obj_id = obj.id
+        expect_id = state.id
```
Comment (Leidysalda) [inline tests/test_models/test_engine/test_db_storage.py:115]: Remember the use of quotes in the comments for python.

File: models/engine/db_storage.py (hunk 1 above)
Comment (juan-skill) [reply models/engine/db_storage.py:83]: Well, Leidy I delete the item variable and I just use the string into objs access

File: tests/test_models/test_engine/test_db_storage.py (hunk 2 above)
Comment (juan-skill) [reply tests/test_models/test_engine/test_db_storage.py:78]: I will delete this function, you all right!

File: tests/test_models/test_engine/test_db_storage.py (hunk 3 above)
Comment (juan-skill) [reply tests/test_models/test_engine/test_db_storage.py:115]: Delete this line code, thank you Leidy

**Pull request merged**